trilitech/jstz
# chore(deps): update octez to proto-alpha

# Context






TL;DR: We discovered that #352 includes DAL host functions for jstz, which aren't available on proto-nairobi versions of octez. 

# Description







The fix is to update to the master version of octez-client/node/smart-rollup-node. This temporarily breaks our deployment to nairobinet -- but we plan to deploy to weeklynet next week, so 🤷‍♂️

# Manually testing the PR



To test you should re-build octez from sources, including the relevant binaries (`octez-client`, `octez-node`, `octez-smart-rollup-node`) in your `octez_path` (specified in the config).  

```sh
make build
cargo run --bin jstz -- sandbox start
cargo run --bin jstz-node -- --kernel-file-path ./logs/kernel.log
cargo run --bin jstz -- deploy examples/counter.js
```
